history-of-the-quakers-v4/original/02-book-6-chapter-15.adoc

```diff
@@ -1035,15 +1027,25 @@ that in the delightful fruition thereof, the Queen may enjoy many days,
 and after a life of comfort, be translated to a glorious immortality.
 Signed on behalf of the said people by
 
+[.signed-section-signature]
 John Field,
 
+[.signed-section-signature]
 Joseph Wyeth.
 
+--
+
+[.embedded-content-document]
+--
+
+[.old-style]
 === The Queen`'s Answer.
```

Comment:
After I left the last comment above, I saw this and remembered why I embedded it....but still not sure if it's the right call.

Reply:
I think this is fine, it's a tough call, doesn't fit perfectly into any category.  This looks good though, so let's go with it.